package.json

```diff
@@ -48,7 +48,8 @@
   },
   "dependencies": {
     "base-64": "^0.1.0",
-    "jsrsasign": "8.0.12",
+    "crypto-js": "^3.1.9-1",
+    "jsbn": "^1.1.0",
     "jwt-decode": "^2.2.0",
```

Comment:
improvement: consider removing `jwt-decode`, add a base64-url-decode method to the base64 class introduced here and do it ourselves